groalea/graphio.py

```diff
@@ -1046,12 +1047,28 @@ def _local2global(self, vid, matrix, color):
                 shape = pgl.Shape(transform4(matrix, shape), pgl.Material(color))
             else:
                 shape = pgl.Shape(transform4(matrix, shape))
-            shape.id = vid
+            shape.id = self._getShapeid(vid)
             final_geometry[vid] = shape
 
         if color:
             colors[vid] = color
 
+    def _getShapeid(self, vid):
+        global msidShapeidDic
+        shapeid = 0
+        print msidShapeidDic
+        print "vid --=", vid

```

Comment:
Remove debug print. If you need it, we can add a log system taht can be activated at runtine.